src/app/project/project-lg-md.component.scss

```diff
@@ -0,0 +1,418 @@
+@import "~assets/styles/base/base.scss";
+
+// medium
+@media (min-width: 768px) {
+
+    .project-info {
+        display: block;
+    }
+
+
+
+
+    /*
+        Project
+    */
+
+
+    .project {
+        background: white;
+
+        .container {
+            padding: 40px 15px 40px;
+        }
+
+    /*
+        PCP banner
+    */
+
+        .banner {
+            background-color: #003366;
+            color: white;
+        }
+
+    /*
+        Side Banner
+    */
+
+        .side-banner {
+
+            height: 100%; /* Full-height: remove this if you want "auto" height */
```

Comment:
Looks good @MaxWardle!

I'm getting some squishing on the side banner on my end still, which causes the scrollbar to display. Looks like the "height: 100%" is overflowing a bit and adding a bunch of blank space below the map. It's good that the scrollbar shows up when this happens, but I wonder if just removing this to auto-size the panel will prevent it for most cases? Works on my end anyway.